packages/node/docker/README.md

```diff
@@ -0,0 +1,42 @@
+**This documentation focuses on the Airnode Docker image and its usage, not the Airnode itself. If you want learn more about the Airnode, please read [its documentation](../README.md).**
+
+# Build
+In order to build Airnode Docker image you need to build the [artifacts image first](../../../docker/README.md). Once you've done that, you can build the Docker image by running following command from the root directory:
+```bash
+docker build -f packages/node/docker/Dockerfile -t api3/airnode:latest .
+```
+
+# Configuration
+## Volumes
+The Airnode needs two configuration files for its run: `config.json` and `secrets.env`. These files need to be passed to the Docker container via volumes.
+
+The Docker container looks for configuration files mounted internally in the `/app/config` directory.
+```bash
+$ tree
+.
+└── config
+    ├── config.json
+    └── secrets.env
+$ docker run -v $(pwd)/config:/app/config ...
+```
+
+# Usage
+Example directory structure and commands for running the Airnode Docker container. The below commands are run from the depicted directory.
+
+> If you are using Windows, use CMD (and not PowerShell), replace `\` with `^` and `$(pwd)` with `%cd%`.
```

Comment:
Why can't I use PowerShell?

Reply:
Don't actually know. I don't have a Windows machine, I was just relying on a wisdom of the person who did the dockerization befor me and added this line :smile:

Reply:
I could possibly try this on windows if there is need for this (I have dual boot) - but I don't have docker installed there so it would be better if someone working on windows already would check this.